tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+  "include": ["lib/**/*", "espree.js"],
+  "compilerOptions": {
```

Comment:
Can we use `"strict": true` in here? https://www.typescriptlang.org/tsconfig#strict

Reply:
Ah, good point

Reply:
I'm not sure it has any effects - we don't have ts code, and didn't enable `"checkJs": true`.

Reply:
Hmm, you could be right, I thought there might be some stricter behavior/validation it might enable. Regardless, it probably wouldn't hurt to use it, especially in case we add `"checkJs": true` later.